### DISCUSSION TITLE
How to use joint in track 1

### DISCUSSION THREAD

**Discussion by Qiandao-Liu** [empriselab/RCareWorld#62] [Help - Programming] [answered]
hi, sorry for interupt, but I spent tons of work try to rotate a joint but it just doesn't work. Could you show me a demo command or what to let a gripper joint rotate for 90 degrees?

**Comment by YoruCathy** [comment]
Could you clarify what do you mean by rotate the gripper joint? Do you want to rotate the end effector itself?

**Reply by Qiandao-Liu** [reply]
Yea it's like to rotate the joint between gripper and arm

**Comment by YoruCathy** [comment, marked as answer]
Rotating the gripper is essentially changing the end effector pose (rotation only). Usually we do it using inverse kinematics rather than modifying the rotation of a single joint. See the new script this commit for reference: https://github.com/empriselab/RCareWorld/commit/2a3bd807a52f37ac72005754d56279fb8755033b
If you are interested, some readings about IK: https://motion.cs.illinois.edu/RoboticSystems/InverseKinematics.html

**Reply by Qiandao-Liu** [reply]
i think i figure out the process, Thanks a lot!
